DATS.json

```diff
@@ -0,0 +1,203 @@
+  "description": "Umbrella dataset of the the Courtois Project on Neuronal Modelling",
+  "identifier": {
+    "@id": "https://w3id.org/dats/schema/identifier_info_schema.json",
+    "@type": "Identifier",
+    "identifier": "https://doi.org/10.5281/zenodo.3875552",
+    "identifierSource": "Zenodo DOI"
+  },
+  "dates": [
+    {
+      "@id": "https://w3id.org/dats/schema/date_info_schema.json",
+      "@type": "Date",
+      "date": "2020-05-04",
+      "type": {
+        "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+        "@type": "Annotation",
+        "value": "Start Date"
+      }
+    }
+  ],
+  "creators": [
+    {
+      "name": "The Courtois Project on Neuronal Modelling"
+    }
+  ],
+  "types": [
+    {
+      "@id": "https://w3id.org/dats/schema/data_type_schema.json",
+      "@type": "DataType",
+      "information": {
+        "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+        "@type": "Annotation",
+        "value": "MRI"
+      }
+    },
+    {
+      "@id": "https://w3id.org/dats/schema/data_type_schema.json",
+      "@type": "DataType",
+      "information": {
+        "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+        "@type": "Annotation",
+        "value": "Basic Demographics"
+      }
+    },
+    {
+      "@id": "https://w3id.org/dats/schema/data_type_schema.json",
+      "@type": "DataType",
+      "information": {
+        "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+        "@type": "Annotation",
+        "value": "BIDS"
+      }
+    }
+  ],
+  "version": "1.1",
+  "availability": "available",
+  "privacy": "Private",
+  "distributions": [
+    {
+    "@id": "http://json-schema.org/draft-04/schema",
+    "@type": "DatasetDistribution",
+    "formats": [
+      "BIDS",
+      "NIfTI",
+      "TSV",
+      "JSON"
+    ],
+    "size" : 1000,
+    "unit" : {
+      "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+      "@type": "Annotation",
+      "value": "GB"
+    },
+    "access": {
+      "landingPage": "https://www.cneuromod.ca/carousel/acquisitions",
+      "authorizations": [
+        {
+          "value": "private"
+        }
+      ]
+    }
+  }
+  ],
+  "isAbout": [
+    {
+      "@id": "https://w3id.org/dats/schema/biological_entity_schema.json",
+      "@type": "BiologicalEntity",
+      "name": "Adult humans"
+    }
+  ],
+  "spatialCoverage": [
+    {
+      "@id": "https://w3id.org/dats/schema/place_schema.json",
+      "@type": "Place",
+      "name": "Québec (province)",
+      "description": "Province of Québec, Canada"
+    }
+  ],
+  "aggregation": "instance of dataset",
+  "dimensions": [
+    {
+      "@id": "https://w3id.org/dats/schema/dimension_schema.json",
+      "@type": "Dimension",
+      "name" : {
+          "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+          "@type": "Annotation",
+          "value": "Structural MRI"
+      },
+      "description": "T1W, T2W, DWI"
```

Comment:
Not sure how detailed this has to be but we have a full list of the structural MRI in  https://docs.cneuromod.ca/en/latest/MRI.html#brain-anatomical-sequences 
including cervical-spinal-cord imaging.

Reply:
@emmetaobrien can we use free text in this description or is there a documented set of available keys anywhere?

Reply:
ok, free text passes validation, so I just used the sequence names found at https://docs.cneuromod.ca/en/latest/MRI.html#brain-anatomical-sequences

Reply:
right now that is free text.